src/atoms/Checkbox/__tests__/Checkbox.native.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Checkbox from './../Checkbox';
+import { renderWithTheme } from '../../../_helpers/testing';
+
+describe('Native <Checkbox />', () => {
+  describe('Unchecked checkxox', () => {
+    test('should render component when not checked with default props', () => {
+      const { container } = renderWithTheme(<Checkbox title="Some Title" />);
+      expect(container).toMatchSnapshot();
+    });
+
+    test('should render component when not checked with help text props', () => {
+      const helpTextContent = 'some Help text';
```

Comment:
rename it to `helpText` do it everywhere

Reply:
changed